packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -699,15 +655,62 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
       setState(() {
         final selection = oldWidget.selectionNotifier.value;
         if (selection.scrollIntoView) {
-          final newPosition = selection.displayRow * defaultRowHeight;
-          // TODO(terry): Optimize selecting row that's in the viewport
-          //              otherwise we jumpTo.
-          scrollController.jumpTo(newPosition);
+          final selectedDisplayRow = selectionRowNumber(selection.node);
```

Comment:
If you can't just do `final selectedDisplayRow = selection.nodeIndex;`, can you please add a comment explaining why? nodeIndex is supposed to be the true index of that row in the table.

Reply:
Its just a node index nothing more.